doc/table_report.py

```diff
@@ -0,0 +1,14 @@
+# %%
+from skrub import TableReport
+from skrub.datasets import fetch_employee_salaries
+
+
+def generate_demo():
+    X = fetch_employee_salaries().X
+    with open("_templates/demo_table_report_generated.html", "w") as f:
+        f.write(TableReport(X).html_snippet())
+
+
+# %%
+generate_demo()
```

Comment:
not a big deal but I think we end up doing it twice because it is done here upon import and then called again in conf.py. you may want to put this line inside a `if __name__ == '__main__'` block

Reply:
typo, I forgot to remove the function call used for debugging